app/src/main/res/layout/state_fragment.xml

```diff
@@ -23,6 +23,7 @@
       android:layout_marginTop="8dp"
       android:layout_marginEnd="8dp"
       android:layout_marginBottom="8dp"
+      android:visibility="gone"
```

Comment:
I suggest removing this line because this will cause the audio related tests in StateFragmentTest to fail.

Reply:
Removed.